### PR TITLE
LPS-169043 move_commons-collections4_to_shared-dependencies

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi Tina, Shuyang,

I am trying to move commons-collections4 to shared-dependencies because we already have a shared-dependencies for org.apache.commons.
Thanks for checking!

### CODE REVIEW

**Comment by catarineaguiar** [comment]
@jeffwu0724 can you rebase, please? thanks

**Comment by jeffwu0724** [comment]
rebased the code

**Comment by jeffwu0724** [comment]
ci:test:sf

**Comment by jeffwu0724** [comment]
ci:test:relevant

**Comment by zhangyan0701** [comment]
No regressions found.

File: modules/apps/batch-engine/batch-engine-service/build.gradle [hunk 1]
```diff
@@ -4,7 +4,7 @@ buildService {
 
 dependencies {
 	compileInclude group: "com.zaxxer", name: "SparseBitSet", version: "1.2"
-	compileInclude group: "org.apache.commons", name: "commons-collections4", version: "4.3"
+	compileInclude group: "org.apache.commons", name: "commons-collections4", version: "4.4"
```
Comment (tinatian) [inline modules/apps/batch-engine/batch-engine-service/build.gradle]: @jeffwu0724 Why you need to bump up the version from 4.3 to 4.4 for a pull that is supposed to move a lib to shared dependencies?

**Comment by jeffwu0724** [comment]
ci:test:relevant

**Comment by jeffwu0724** [comment]
ci:test:sf

File: modules/apps/batch-engine/batch-engine-service/build.gradle (hunk 1 above)
Comment (jeffwu0724) [reply modules/apps/batch-engine/batch-engine-service/build.gradle]: When I add "commons-collections4" to shared-dependencies, I add it with version 4.4.
The reason I choose 4.4 here is that, in our code base, we are using 4.2, 4.3, and 4.4 at the same time. I decided to choose the highest version for easier maintenance in the future.
That's why I add "org.apache.commons:commons-collections4:4.4" in source-formatter, and auto sf at the end to make sure everyone is using version 4.4

**Comment by Kvale1733** [comment]
@jeffwu0724 No regressions found.

**Pull request closed without merging**